src/utils/helpers.ts

```diff
@@ -21,3 +23,17 @@ export function exhaustiveCheck (_value: never): never {
 
 export const geojsonPointToLongitude = (point: Point): number => point.coordinates[0]
 export const geojsonPointToLatitude = (point: Point): number => point.coordinates[1]
+
+export function compareAreaLeftRightIndex (a: AreaType, b: AreaType): number {
```

Comment:
I considered merging these two, but then I'd need to be able to narrow the type of `a` and `b`. And that's not very easy since these are custom types, not just strings or numbers. And so it gets a bit messy. I figure just splitting them out is a bit repetitive but a lot more readable.